### PR TITLE
[DC-3073] the notebook, `analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report4_dateshift.py` needs several updates

### PR DESCRIPTION
**Pull request by ratuagga**
updated dateshift notebook for errors.

### CODE REVIEW

**Review by lledezma — CHANGES_REQUESTED** [review]
Your updates cover the scope of the ticket, but one update causes `Q14` check to fail. Also,
`Q14` results are appended as `Query13` and `Q15 `results are appended as `Query14.3` to the final results table, and this is misleading. You should update those dictionary key values in lines `396`, `399`, `497`, and `500`.

**Comment by ratuagga** [comment]
> Your updates cover the scope of the ticket but one update causes `Q14` check to fail. Also,
> `Q14` results are appended as `Query13` and `Q15 `results are appended as `Query14.3` to the final results table, and this is misleading. You should update those dictionary key values in lines `396`, `399`, `497`, and `500`.

Nice catch. thanks

File: data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report4_dateshift.py [hunk 1]
```diff
@@ -386,8 +386,8 @@
 
 query = JINJA_ENV.from_string("""
 SELECT COUNT (*) AS n_row_not_pass
-FROM  `{{project_id}}.{{pipeline}}.pid_rid_mapping`
-WHERE shift <=0
+FROM  `{{project_id}}.{{pipeline}}.primary_pid_rid_mapping`
+WHERE shift BETWEEN 1 AND 364
 
 """)
```
[the remark this answers is not in the document]
Comment (ratuagga) [reply data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report4_dateshift.py:392]: Yes, I wanted to check that with Lauren as this range for shift was given in the ticket and these many rows not passing means either the query need update or there is something else that needs to be checked.
I'll let you know how that goes.

**Review by lledezma — APPROVED** [review]
Looks good!

**Pull request merged**